cmd/kubeadm/app/apis/kubeadm/v1beta2/doc.go

```diff
@@ -0,0 +1,280 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=kubeadm.k8s.io
+// +k8s:deepcopy-gen=package
+// +k8s:conversion-gen=k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm
+
+// Package v1beta2 defines the v1beta2 version of the kubeadm configuration file format.
+// This version graduates the configuration format to BETA and is a big step towards GA.
+//
+//A list of changes since v1alpha3:
+//	- "apiServerEndpoint" in InitConfiguration was renamed to "localAPIEndpoint" for better clarity of what the field
+//	represents.
+//	- Common fields in ClusterConfiguration such as "*extraArgs" and "*extraVolumes" for control plane components are now moved
+//	under component structs - i.e. "apiServer", "controllerManager", "scheduler".
+//	- "auditPolicy" was removed from ClusterConfiguration. Please use "extraArgs" in "apiServer" to configure this feature instead.
+//	- "unifiedControlPlaneImage" in ClusterConfiguration was changed to a boolean field called "useHyperKubeImage".
+//	- ClusterConfiguration now has a "dns" field which can be used to select and configure the cluster DNS addon.
+//	- "featureGates" still exists under ClusterConfiguration, but there are no supported feature gates in 1.13.
+//	See the Kubernetes 1.13 changelog for further details.
+//	- Both "localEtcd" and "dns" configurations now support custom image repositories.
+//	- The "controlPlane*"-related fields in JoinConfiguration were refactored into a sub-structure.
+//	- "clusterName" was removed from JoinConfiguration and the name is now fetched from the existing cluster.
+//
+// Migration from old kubeadm config versions
+//
+// Please convert your v1alpha3 configuration files to v1beta2 using the "kubeadm config migrate" command of kubeadm v1.13.x
```

Comment:
we'll need to have the doc.go as a TODO.
the search and replace is fine for now.